string cad_item

---------------
bool success
geometry_msgs/Pose target_pose
geometry_msgs/Pose corner_pose
geometry_msgs/Pose original_pose
cad_connection[] connection